winforms/tests_backend/icons.py

```diff
@@ -35,5 +35,5 @@ def assert_default_icon_content(self):
             == Path(toga_winforms.__file__).parent / "resources/toga.ico"
         )
 
-    def assert_platform_icon_content(self):
-        assert self.icon._impl.path == self.app.paths.app / "resources/logo-windows.ico"
+    def assert_platform_icon_content(self, platform):
```

Comment:
This shouldn't be required. The testbed backend is, by definition, platform specific. There isn't a separate FreeBSD backend, so there's no need for configurability at this level.

*At best*, there's a need for the GTK backend to differentiate between Linux and FreeBSD icons - but that can be entirely contained in the GTK test backend. 

Even then, I'm not 100% convinced that's the right approach. The icons are going to be interpreted by GTK, not BSD; it's arguable whether the icons for linux should be suffixed "-gtk" rather than "-linux", specifically because of this.

Reply:
I'd say a potential conflict with Qt is much more likely than "GTK on Windows/macOS". 

I could even see an argument for supporting both suffixes - `-<platform>` and `-gtk` - on the basis that you *might* want freeBSD-specific icons on the GTK backend to match FreeBSD desktop style (I don't know if that's a thing that is likely in practice, but it seems plausible).

However, as you note, this is probably best handled as a separate discussion/PR.